consts/errors.go

```diff
@@ -9,26 +9,27 @@ import (
 )
 
 const (
-	MsgErrInsertUser         string = "unable to insert new user into db:"
-	MsgErrInsertToken        string = "unable to insert token into db:"
-	MsgErrGeneratingUUID     string = "generating uuid:"
-	MsgErrGeneratingToken    string = "generating token:"
-	MsgErrEmailRequest       string = "unable to make email request object:"
-	MsgErrSendEmail          string = "unable to send email:"
-	MsgErrDeleteUser         string = "unable to delete user:"
-	MsgErrGetUserRow         string = "unable to get user row:"
-	MsgErrUpdateUserRow      string = "unable to update user row:"
-	MsgErrAuthenticateUser   string = "failed to authenticate user:"
-	MsgErrMatchPassword      string = "failed to match password:"
-	MsgErrMatchEmail         string = "email does not match"
-	MsgErrSecret             string = "failed to insert new secret into db:"
-	MsgErrGetActiveSecret    string = "failed to get active secret row from db:"
-	MsgErrLookUpActiveSecret string = "failed to look up active secret from db"
-	MsgErrInsertingJWToken   string = "failed to insert jwt into db:"
-	MsgErrGetExistingToken   string = "error retrieving existing token:"
-	MsgErrPermissionMismatch string = "permission level does not match"
-	MsgErrValidatingIdentity string = "failed to validate identity:"
-	MsgErrValidatingToken    string = "failed to match token with db:"
+	MsgErrInsertUser                string = "unable to insert new user into db:"
+	MsgErrInsertToken               string = "unable to insert token into db:"
```

Comment:
What type of token is inserted?

Reply:
Honestly, this user svc can be sending email other than confirming registration purposes.